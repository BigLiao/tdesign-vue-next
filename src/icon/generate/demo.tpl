<template>
  <!--
    该文件有脚本 tdesign/tdesign-web-vue/src/icon/generate/index.js 自动生成。
    若需要编辑，请编辑源文件：tdesign/tdesign-web-vue/src/icon/generate/demo.tpl 。
  -->
  <div class="t-demo-iconfont">
    <p>目前设计师给到的全部图标</p><br>
    <%= icons %>
  </div>
</template>

<style lang="less">
  .t-demo-iconfont {
    .t-demo-icon {
      width: 130px;
      height: 100px;
      margin-right: 24px;
      display: inline-block;
      text-align: center;
      vertical-align: top;
      .t-icon {
        font-size: 25px;
        margin-right: 0;
        color: #999999;
      }
      .t-icon:hover {
        color: #0052d9;
        font-size: 30px;
      }
      .t-demo-icon-name {
        text-align: center;
        margin-top: 8px;
        font-size: 12px;
      }
    }
  }
</style>
