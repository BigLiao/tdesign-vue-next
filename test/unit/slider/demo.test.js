import { mount } from '@vue/test-utils';
import demo from '@/examples/slider/demos/base.vue';

// unit test for component in examples.
describe('Slider', () => {
  it('base demo works fine', () => {
    const wrapper = mount(demo);
    expect(wrapper.element).toMatchSnapshot();
  });
});
