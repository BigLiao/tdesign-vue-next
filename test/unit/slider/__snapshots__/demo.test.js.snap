// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Slider base demo works fine 1`] = `
<div
  class="demo-slider"
>
  <div
    class="block"
  >
    <div
      aria-valuetext="12"
      class="t-slider-contianer"
      modelvalue="12"
    >
      <div
        aria-disabled="false"
        aria-orientation="horizontal"
        aria-valuemax="100"
        aria-valuemin="0"
        class="t-slider"
        role="slider"
      >
        <div
          class="t-slider__rail"
        >
          <div
            class="t-slider__track"
            style="width: 0%; left: 0%;"
          />
          <div
            class="t-slider__button-wrapper"
            disabled="false"
            show-tooltip="true"
            style="left: 0%;"
            tabindex="0"
          >
            <div
              class="t-popup-reference"
              popper-class=""
            >
              <transition-stub>
                <div
                  aria-hidden="true"
                  class="t-popup"
                  role="tooltip"
                  style="display: none;"
                >
                  <div
                    class="t-popup-content t-popup-content--arrow"
                    style="max-width: 480px;"
                  >
                    0
                    <div
                      class="t-popup__arrow"
                      data-popper-arrow="true"
                    />
                  </div>
                </div>
              </transition-stub>
              
              <div
                class="t-slider__button"
              />
              
            </div>
          </div>
          <!---->
          <!---->
          <!---->
        </div>
      </div>
      <div
        class="t-slider-input-container"
      >
        <!---->
        <!---->
        <!---->
      </div>
    </div>
  </div>
  <div
    class="block"
  >
    <div
      aria-valuetext="30-70"
      class="t-slider-contianer"
      modelvalue="30,70"
    >
      <div
        aria-disabled="false"
        aria-orientation="horizontal"
        aria-valuemax="100"
        aria-valuemin="0"
        class="t-slider"
        role="slider"
        tooltip-props="[object Object]"
      >
        <div
          class="t-slider__rail"
        >
          <div
            class="t-slider__track"
            style="width: 0%; left: 0%;"
          />
          <div
            class="t-slider__button-wrapper"
            disabled="false"
            show-tooltip="true"
            style="left: 0%;"
            tabindex="0"
          >
            <div
              class="t-popup-reference"
              popper-class=""
            >
              <transition-stub>
                <div
                  aria-hidden="true"
                  class="t-popup"
                  role="tooltip"
                  style="display: none;"
                >
                  <div
                    class="t-popup-content t-popup-content--arrow"
                    style="max-width: 480px;"
                  >
                    0
                    <div
                      class="t-popup__arrow"
                      data-popper-arrow="true"
                    />
                  </div>
                </div>
              </transition-stub>
              
              <div
                class="t-slider__button"
              />
              
            </div>
          </div>
          <div
            class="t-slider__button-wrapper"
            disabled="false"
            show-tooltip="true"
            style="left: 0%;"
            tabindex="0"
          >
            <div
              class="t-popup-reference"
              popper-class=""
            >
              <transition-stub>
                <div
                  aria-hidden="true"
                  class="t-popup"
                  role="tooltip"
                  style="display: none;"
                >
                  <div
                    class="t-popup-content t-popup-content--arrow"
                    style="max-width: 480px;"
                  >
                    0
                    <div
                      class="t-popup__arrow"
                      data-popper-arrow="true"
                    />
                  </div>
                </div>
              </transition-stub>
              
              <div
                class="t-slider__button"
              />
              
            </div>
          </div>
          <!---->
          <!---->
        </div>
      </div>
      <div
        class="t-slider-input-container"
      >
        <!---->
        <!---->
        <!---->
      </div>
    </div>
  </div>
</div>
`;
